src/UI Component/Adminpanel/main.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import ViewVerified from "./Activities/verifyloan/verifyloan";
+import CurrentLoan from "./Activities/currentloan";
+import AllLoanApplications from "./Activities/allloanapplication";
+import FirstPane from "../User/firstpane";
+import AdminHome from "./adminhome";
+import LoanApprovalPage from "./Activities/loanapproval"
+import SpecialLoanRequest from "./Activities/specialloans.js";
+import PostTransactionForm from "./Activities/postloanrepayment/posttransactionform";
+
+const Button = ({ name, activeButton, onClick }) => (
+  <button
+    className={`bg-blue-400 font-bold rounded-lg text-black px-4 py-2 mr-4 ${
+      activeButton === name && "bg-blue-700"
+    }`}
+    onClick={() => onClick(name)}
+  >
+    {name}
+  </button>
+);
+
+const MainPage = () => {
+  const [activeButton, setActiveButton] = useState("adminHome");
```

Comment:
Why are we defaulting to adminHome here?

Reply:
This sets the default active button to "adminHome" when the component is first rendered, meaning that when the page loads the "Admin Home" section is displayed first